js/packages/cli/src/candy-machine-cli.ts

```diff
@@ -291,14 +291,19 @@ programCommand('verify_token_metadata')
     const { number } = cmd.opts();
 
     const startMs = Date.now();
-    log.info('started at: ' + startMs.toString());
+    log.info(
+      `\n==> Starting verification: ${
+        new Date(startMs).toString().split(' G')[0]
+      }`,
+    );
     verifyTokenMetadata({ files, uploadElementsCount: number });
 
     const endMs = Date.now();
     const timeTaken = new Date(endMs - startMs).toISOString().substr(11, 8);
     log.info(
-      `ended at: ${new Date(endMs).toString()}. time taken: ${timeTaken}`,
+      `==> Verification ended: ${new Date(endMs).toString().split(' G')[0]}`,
     );
+    log.info(`Elapsed time: ${timeTaken}\n`);
```

Comment:
Ditto above .. also please don't duplicate the date _prettyfing_ code pull out into a separate function or use something like [date-time](https://github.com/sindresorhus/date-time)